src/commands/dev/dev.mjs

```diff
@@ -248,6 +252,7 @@ export const createDevCommand = (program) => {
         .argParser((value) => Number.parseInt(value))
         .hideHelp(true),
     )
+    .addOption(new Option('--no-open', 'disables the automatic opening of a browser window'))
```

Comment:
This is already possible by setting the `BROWSER` environment variable to `none`. Not against adding this flag, just wondering how this is related to the monorepo work?

Reply:
I was so annoyed by testing the dev and always having the browser opened and I thought a flag is handier than the env var